packages/compass-aggregations/src/modules/env.js

```diff
@@ -18,21 +12,6 @@ export const INITIAL_STATE = ON_PREM;
  *
  * @returns {String} The new state.
  */
-export default function reducer(state = INITIAL_STATE, action) {
-  if (action.type === ENV_CHANGED) {
-    return action.env;
-  }
+export default function reducer(state = INITIAL_STATE) {
```

Comment:
If you are looking at this and thinking to yourself "Huh, this doesn't do much anymore, do we need it in the store at all?", that's a very good question to ask. Technically the way we store these values that should never be changed in the store is probably not the best way to handle it, but a lot of aggregation code relies on these values to be here (especially combined with the prop drilling we are doing everywhere in this plugin), so to avoid a massive refactor, I am keeping reducers as-is, it's not the worst thing in the world, just a unnecessarily verbose way to define something like this